mantidproject/mantid
# Single function to load parameters associated with a nexus file

Fixes #12940 

To test:
(1) Review Code
(2) Check the unit tests are adequate.
(3) Check that mantid loads some nexus files with parameters associated with them as normal. No change should have occurred.
